packages/x-charts-pro/src/Heatmap/HeatmapTooltip.tsx

```diff
@@ -39,9 +39,6 @@ const useUtilityClasses = (ownerState: { classes: HeatmapTooltipProps['classes']
   return composeClasses(slots, getChartsTooltipUtilityClass, classes);
 };
 
-/**
- * @ignore - internal component.
- */
```

Comment:
Even if it's applied to a different component, the `shouldSkip` logic triggers before we ever try to read the jsdocs of the components (to be faster), so we have to remove this. to get the API page of `HeatmapTooltip` to generate.